release_notes/release_notes.html

```diff
@@ -1,6 +1,6 @@
 <b>Parser Release Notes - Published by Splunk January 25, 2022</b>
 <br><br>
-<b>Version 2.5.7 - Released January 25, 2022</b>
+<b>Version 2.5.8 - Released January 25, 2022</b>
```

Comment:
Could you run 
```
pre-commit install
pre-commit autoupdate
pre-commit run --all-files
```
once? 
When I run these just now, I notice some wheel changes.

Reply:
For reference, as we discussed, it's likely that some newer version of a sub-dependency was released (the direct dependency probably specified >= min version for it)